web/client/components/map/openlayers/__tests__/Layer-test.jsx

```diff
@@ -285,6 +285,30 @@ describe('Openlayers layer', () => {
+            },
+            "url": 'https://sample.com/geoserver/wms?SERVICE=WMS&VERSION=1.3.0&REQUEST=GetMap&FORMAT=image%2Fvnd.jpeg-png8&TRANSPARENT=true&LAYERS=workspace%3Agdy_pet_sys&STYLES=&SRS=EPSG%3A4326&CRS=EPSG%3A4326&TILED=false&authkey=9cf6ff4e-2529-40cb-9ff8-f142c550b579&WIDTH=1410&HEIGHT=497&BBOX=17.68798828125%2C-78.2666015625%2C39.52880859375%2C-16.3037109375'
+        };
+        // create layers
+        let layer = ReactDOM.render(
+            <OpenlayersLayer type="wms"
+                options={options} map={map}/>, document.getElementById("container"));
+
+
+        expect(layer).toExist();
+        // count layers
+        expect(map.getLayers().getLength()).toBe(1);
```

Comment:
I don't see any expect that checks what we are changing with this PR

Reply:
I think the only meaningful test is to check we are sending a request with POST method